README.md

```diff
@@ -18,11 +18,6 @@
   - [Containers](#containers)
   - [Installation from source](#installation-from-source)
 - [Usage](#usage)
-  - [Population Reference Graphs](#population-reference-graphs)
```

Comment:
Maybe add a link in the TOC to the toy example also?

Reply:
done